### PR TITLE
Add Week one assessment & small adjustments to week one lectures

### PR DESCRIPTION
**Pull request by AmberWilkie**

### CODE REVIEW

File: miscellaneous/assessments/week_1_assessment.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+Create a new folder called 'cars' and then a file called 'my_cars.rb'  
+Add two hashes, one for each `car`, with the following attributes: `wheels`, `max_speed`, `color`  
+Create an array that contains both cars.  
+How do we use the array to access the second car? How do we find the second car's `color`?
+
+## Question 2 - Classes and Methods
+Create a new file called 'car.rb' with:
+ - a class Car
+ - a method that "paints" a car a new color.
+ - a method that checks if the current car is the first car (`is_first_car?`)
```
Comment (diraulo) [inline miscellaneous/assessments/week_1_assessment.md]: Sorry for late comment on this but @AmberWilkie, how do you expect them to check for this?

**Comment by AmberWilkie** [comment]
I was looking for them to use an evaluation, like
def is_first_car?(car)
car == car_array[0]
end


*Amber Wilkie*
www.amberwilkie.com
0720.08.27.07 | Instagram <http://www.instagram.com/amberwphoto>

**Comment by diraulo** [comment]
OK @AmberWilkie. I guess this method isn't part of the class `Car` then. Wasn't quite clear while reading the question.

**Comment by AmberWilkie** [comment]
Yeah you'd still need access to the previous arrays. I'll make that clear.


*Amber Wilkie*
www.amberwilkie.com
0720.08.27.07 | Instagram <http://www.instagram.com/amberwphoto>

**Pull request merged**